cpp/ql/lib/semmle/code/cpp/models/implementations/Scanf.qll

```diff
@@ -70,3 +67,40 @@ private class SscanfModel extends ArrayFunction, TaintFunction, AliasFunction, S
       ]
   }
 }
+
+/**
+ * The standard function `scanf` and its assorted variants
+ */
+private class ScanfModel extends ScanfFunctionModel, LocalFlowSourceFunction {
+  ScanfModel() { this instanceof Scanf }
```

Comment:
Might as well be nice and help out to the compiler 😂
```suggestion
private class ScanfModel extends ScanfFunctionModel, LocalFlowSourceFunction instanceof Scanf {
```

Reply:
Fixed

Reply:
> We really need a QL-for-QL check for this.

Agreed 👍